server/events/vcs/testdata/github-branch-protection-require-linear-history.json

```diff
@@ -0,0 +1,6 @@
+{
+    "url": "https://api.github.com/repos/octocat/Hello-World/branches/master/protection",
+    "required_linear_history": {
+        "enabled": true
+    }
+}
```

Comment:
Mind adding a new line here? We probably need a gitattributes in this repo. If you can add one too for .go and .json that would be nice too

Reply:
IDK why but VSCode automatically removes a newline on save for the json file
so I guess this is a "accepted pattern"?

Reply:
No worries. We can automate this away using https://github.com/runatlantis/atlantis/pull/3244